cli/lsp/config.rs

```diff
@@ -859,9 +859,6 @@ impl Config {
     specifier: &ModuleSpecifier,
   ) -> Option<&LanguageWorkspaceSettings> {
     let workspace_settings = self.workspace_settings_for_specifier(specifier);
-    if specifier.scheme() == "deno-notebook-cell" {
-      return Some(&workspace_settings.typescript);
-    }
```

Comment:
Is there a substitute for this? I think the inferred media type will be `Unknown` and it won't respect `typescript.*` workspace settings.

Reply:
Good catch! Forgot about that. Added a `.ts` extension to the specifier (which works fine even if there's a .ts file at the notebook file path with a `.ts` extension)